tailwind.config.cjs

```diff
@@ -26,7 +26,8 @@ module.exports = {
         lightSlate:"#11181C",
         gradFirst: "#FF932F",
         gradMiddle: "#FF3C3C",
-        gradLast: "#FF3C3C"
+        gradLast: "#FF3C3C",
```

Comment:
where are these new variables used and how was their naming decided?

Reply:
Those variables are used to create a gradient color for the header on the hero section. Whereas 'gradFirst' is the start, and 'gradLast' is the ending color of the gradient.

Reply:
#230 ?

Reply:
Nope, sir! That's for the header text! 
![Screenshot 2022-07-29 at 9 01 04 PM](https://user-images.githubusercontent.com/48399373/181788224-07062500-ab3f-483d-9f46-3c00094902fb.png)

Reply:
FWIW, the grad shortening is not entirely clear until you spelled it out. I think I will open an issue to update that to state "gradientFirst" etc.

Not blocking for this, but we should address that.

Reply:
Making the gradient text classes more intuitive would be nice:
- `headerTextGradientStart/headerTextGradientMiddle/headerTextGradientEnd` - 👎 
- `headerTextGradient0/headerTextGradient50/headerTextGradient100` - 👍 

However, this is the wrong approach as we do not have the colors available as (actually) re-usable classes, we should create the gradient based on existing theme colors, rather than create duplicate color classes subsets for the gradients. 

Visually, our start, middle and end color colors are not perceived by our users, it is unlikely the end and start color stops need additional hex colors other than the ones we have. I'm keen to stop introducing unwanted things we then have to open new fix PRs for.

Reply:
open this issue https://github.com/open-sauced/hot/issues/266

That change should happen after this PR.